src/Core/Components/DataGrid/Columns/SelectColumn.cs

```diff
@@ -209,6 +209,9 @@ public DataGridSelectMode SelectMode
     [Parameter]
     public override IGridSort<TGridItem>? SortBy { get; set; }
 
```

Comment:
Please add a comment here. (we use this for the documentation in the demo site).